src/content/developers/docs/mev/index.md

```diff
@@ -10,19 +10,19 @@ Maximal extractable value (MEV) refers to the maximum value that can be extracte
 
 ### Miner extractable value
```

Comment:
```suggestion
### Maximal extractable value {#maximal-extractable-value}
```

Does it make sense to change this header? I think this needs the anchor tag as well.